site/sitecontent/terms.md

```diff
@@ -0,0 +1,264 @@
+
+## Account Terms
+
+When using the Service to host projects, you are responsible for maintaining the security of your account and any projects, and you are fully responsible for all activities that occur under the account and any other actions taken in connection with projects.
+
+Your account must not be named in a manner that misleads other users of the Service into thinking that you are another person, organisation or company.
+
+You must not describe or assign keywords to projects in a misleading or unlawful manner, including in a manner intended to trade on the name or reputation of others, and FOSSi Foundation may change or remove any description or keyword that it considers inappropriate or unlawful, or otherwise likely to cause FOSSi Foundation liability. You must immediately notify FOSSi Foundation of any unauthorized uses of your projects, your account or any other breaches of security. FOSSi Foundation will not be liable for any acts or omissions by You, including any damages of any kind incurred as a result of such acts or omissions.
+
+## Responsibility of Contributors
+
+If you maintain a project, contribute to a project, post material to the Service, post links on the website, or otherwise make (or allow any third party to make) material available by means of the Service (any such material, “Content”), You are entirely responsible for the content of, and any harm resulting from, that Content. That is the case regardless of whether the Content in question constitutes text, graphics, a computer-aided design or computer-aided manufacturing file, an audio file, computer software or a programmable logic bitstream. By making Content available, you represent and warrant that:
+
+* the downloading, copying and use of the Content will not infringe the proprietary rights, including but not limited to the copyright, patent, trademark or trade secret rights, of any third party;
+
+* if your employer has rights to intellectual property you create, you have either (i) received permission from your employer to post or make available the Content, including but not limited to any software, or (ii) secured from your employer a waiver as to all rights in or to the Content;
+
+* you have fully complied with any third-party licences relating to the Content, and have done all things necessary to successfully pass through to end users any required terms;
+
+* the Content does not contain or install any viruses, worms, malware, Trojan horses or other harmful or destructive content;
+
+* the Content is not spam, is not machine- or randomly-generated, and does not contain unethical or unwanted commercial content designed to drive traffic to third party sites or boost the search engine rankings of third party sites, or to further unlawful acts (such as phishing) or mislead recipients as to the source of the material (such as spoofing);
```

Comment:
"machine-generated"

This is very hard to apply to code. Any project using a scaffolding tool, or some fancy tool that produces some code from the spec would be disqualified

Reply:
True, it can be misunderstood. Do you think we should remove it or better specify what is meant?